### DISCUSSION TITLE
Consolidate 2 options pane to one

### DISCUSSION THREAD

**Discussion by heejaechang** [Microsoft/PTVS#7337] [Enchancements]
VS currently has 2 places for python options

one in Text Editor->Python
![image](https://user-images.githubusercontent.com/1333179/182442229-2466170e-d915-4a9a-af82-fff25b0fa030.png)

one at the root
![image](https://user-images.githubusercontent.com/1333179/182442301-9188a1b5-673d-4107-9a9f-6d2b4d8b653a.png)

It is confusing which location contains options I am looking for. for example, one option says "interactive window"
![image](https://user-images.githubusercontent.com/1333179/182442665-4aabadba-aca1-4d34-9726-a74d275ed744.png)

and the other option page contains different options for REPL
![image](https://user-images.githubusercontent.com/1333179/182442894-cbee755f-2853-4139-8852-30016db1ee85.png)

it would be nice to have all these options scattered consolidated to one place.

**Comment by StellaHuang95** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
